diff --git a/ios/DevMenuViewController.swift b/ios/DevMenuViewController.swift
index 5d58874ccc74167019f769398eff1bd4ea39e9af..78d21715d178463b56dfc9757f82e1852265a2f6 100644
--- a/ios/DevMenuViewController.swift
+++ b/ios/DevMenuViewController.swift
@@ -63,7 +63,11 @@ class DevMenuViewController: UIViewController {
   // MARK: private
 
   private func initialProps() -> [String: Any] {
-    let isSimulator = TARGET_IPHONE_SIMULATOR > 0
+    #if targetEnvironment(simulator)
+    let isSimulator = true
+    #else
+    let isSimulator = false
+    #endif
 
     return [
       "showOnboardingView": manager.shouldShowOnboarding(),
